client/src/components/CreatePlantForm.jsx

```diff
@@ -0,0 +1,345 @@
+import PropTypes from 'prop-types'
+
+const CreatePlantForm = ({
+
+  handleSubmit, onSubmit, register, formLabelStandardStyle  }) => {
+  return (
```

Comment:
I wonder if you'd benefit from splitting this file up? by having parts of the form in separate files, so this main file is easier to read (it's hard to picture in my mind what it's rendering, though your screenshot was very helpful for this). Think a year from now when you want to go back to this and make some changes. The cognitive load of interpreting this file will be higher than if it's split into sub parts. Not a hard rule, really, just a recommendation (I've been that person coming back to the file months after and wishing It wasn't 300 lines long)

Reply:
Good point. I'll group relevant fields into separate components and then import them.